### ISSUE TITLE
API - limit parameter does not change the number of sentences returned

### ISSUE THREAD

**Issue by DerekDuchesne** [Tatoeba/tatoeba2#2983]
1. Go to https://tatoeba.org/en/api_v0/search?query=%22%E5%AD%A6%E6%A0%A1%22&from=jpn&to=eng&page=1&limit=100

Expected behavior:
I expected the number of results returned to be 100, but it was 10. Strangely, the perPage property does show 100, but the actual number of results is always 10.

**Comment by Yorwba** [comment]
You're right, [the limit is hardcoded to 10](https://github.com/Tatoeba/tatoeba2/blob/5077a5573b3e65ebfb4cfe7011e0778cd4ef02f7/src/Controller/ApiV0Controller.php#L80).

The rationale was explained in [a comment on the original issue about adding an API](https://github.com/Tatoeba/tatoeba2/issues/2669#issuecomment-860283068). Basically, Trang was concerned that returning too many results would cause performance issues.

Until we decide to lift this restriction, you'll need to request ten pages with 10 results each to get the 100 you want. (I guess that might also lead to performance issues... but it would at least let you get things done.)

**Comment by DerekDuchesne** [comment]
@Yorwba Thank you for the explanation! I'd prefer if the limit was 100 to make pagination more efficient, but I understand now that it's not a bug. Should I close the issue or keep it open and change the label from Bug to Feature to see if there's interest in raising the limit?

**Comment by Yorwba** [comment]
There's at least one bug here, namely that setting the `limit` parameter in the request changes the value in the returned JSON even though it doesn't actually have the intended effect. So I think this issue can stay as-is.

**Comment by jiru** [comment]
Closing because the old API is now deprecated. This issue can be solved by [switching](https://en.wiki.tatoeba.org/articles/show/api-migration-v1) to the [new API](https://api.tatoeba.org/).
